vizro-core/docs/pages/user_guides/custom_charts.md

```diff
@@ -65,13 +70,19 @@ Building on the above, there are several routes one can take. The following exam
                 ),
             ],
             controls=[
-                vm.Filter(column="petal_width"),
+                vm.Parameter( # (2)!
+                    targets=["enhanced_scatter.hline"],
+                    selector=vm.Slider(min=2, max=5, step=1, value=3, title="Horizontal line"),
+                ),
             ],
         )
         dashboard = vm.Dashboard(pages=[page_0])
 
         Vizro().build(dashboard).run()
         ```
+
+        1.  Note that arguments of the custom chart can be parametrized. Here we choose to parametrize the `hline` argument (see below).
+        2.  Here we parametrize the `hline` argument, but any other argument can be parametrized as well. Since there is complete flexibility regarding what can be derived from such arguments, the dashboard user has a wide range of customization options.
```

Comment:
Seems like repetition looking at the second point? Feel free to ignore though if you think this is more helpful
```suggestion
        1.  Note that arguments of the custom chart can be parametrized.
        2.  Here we parametrize the `hline` argument, but any other argument can be parametrized as well. Since there is complete flexibility regarding what can be derived from such arguments, the dashboard user has a wide range of customization options.
```

Reply:
The thing is, since those things are clickable, they do not appear at the same time, so I think the repetition is fine, especially if someone just opens one of them